tests/helpers/test_config_validation.py

```diff
@@ -309,6 +309,36 @@ def test_time_period():
     assert -1 * timedelta(hours=1, minutes=15) == schema('-1:15')
 
 
+def test_time_period_with_unit():
+    """Test time_period_str_unit validation."""
+    # Test with time_period to see if it plays nicely with other formats
+    schema = vol.Schema(cv.time_period)
+
+    assert timedelta(seconds=70) == schema('70')
```

Comment:
assert order is the other way around. Value vs expected.

Reply:
I agree, it makes much more sense the other way. For context: Did it this way because other tests did it this way, but no need to repeat past mistakes 👍